coursefinder/static/js/finder-summary.js

```diff
@@ -55,6 +57,14 @@
             }
         },
 
+        getSubjectName: function(item) {
+            if (location.href.indexOf('/cy/') === -1) {
+                return item.englishname;
```

Comment:
Inconsistent syntax `englishname ` and `welsh_name`, should it not be `english_name`?

Reply:
Possibly, but it's following the format of the subject JSON file.
I can update the current JSON file, but whatever produced it will need updating too.

Reply:
I've updated the script (in repo alpha-scripts), I'll move the script across and place under dev_tools.

If you can change the variable name in subjects.json and then the item name that'll be good

Reply:
Variable name changed.